meta-iotqa/scripts/contrib/licensetree-whitelist.txt

```diff
@@ -0,0 +1,35 @@
+virtual/update-alternatives
+make
+libtool-cross
+ptest-runner-dev
+ptest-runner
+virtual/gettext
+perl
+base-files
+cryptodev-linux
+shadow
+shadow-sysroot
+update-rc.d
+initscripts
+bash-completion
+iptables
+nftables
+ppp
+wpa-supplicant
+cpp
+ofono
+bluez5
+connman
+util-linux
+udev
+libgcc-initial
+linux-libc-headers
+python-core
+python3-core
+kernel-4.9.20-yocto-standard
```

Comment:
The dependencies through linking are known to bitbake and are used to add the necessary RDEPENDS. Perhaps instead of using the final RDEPENDS, one could take the information about linking? Then there would be no need for a whitelist.

However, the information might not be readily available and an enhancement request might be needed.